views/013_check_summary.sql

```diff
@@ -159,4 +159,15 @@ CREATE MATERIALIZED VIEW IF NOT EXISTS check_size_summary as
     agg_check_statuses.size / count AS avg_size
   FROM
     agg_check_statuses
-    JOIN checks ON checks.id = agg_check_statuses.check_id;
\ No newline at end of file
+    JOIN checks ON checks.id = agg_check_statuses.check_id;
+
+-- check_summary_for_config
```

Comment:
Maybe `check_summary_by_config` can be removed in favor of this ...?

Reply:
@aditya yeah, it can